packages/SwingSet/src/liveslots/liveslots.js

```diff
@@ -1096,6 +1123,14 @@ function build(
       assert(exportedVPIDs.has(vpid), vpid);
       const rc = resolutionCollector();
       const resolutions = rc.forPromise(vpid, isReject, value);
+      try {
+        resolutions.forEach(([_xvpid, _isReject, resolutionCD]) => {
```

Comment:
if we wind up with multiple copies of this `forEach` pattern, maybe we should have a helper function like `checkResolutionSize(resolutions, syscall.exit)`

Reply:
We have two.